test_client/src/com/owncloud/android/lib/test_project/TestActivity.java

```diff
@@ -338,11 +336,6 @@ public RemoteOperationResult removeShare(int idShare) {
 		
 	}
 
-    public RemoteOperationResult getQuota() {
-        RemoteGetUserQuotaOperation getUserQuotaOperation = new RemoteGetUserQuotaOperation();
-        return getUserQuotaOperation.execute(mClient);
-    }
-	
```

Comment:
We could do a mockup test case because we don't have real servers with various versions and sample data deployed. But test project and sample client never really worked. Maybe sometime in the future I can work on automating server deployment and testing and all that stuff.